cdap-app-fabric/src/main/java/io/cdap/cdap/internal/app/worker/sidecar/ArtifactLocalizerTwillRunnable.java

```diff
@@ -100,7 +103,13 @@ public static Injector createInjector(CConfiguration cConf, Configuration hConf)
 
     modules.add(new ConfigModule(cConf, hConf));
     modules.add(new IOModule());
-    modules.add(RemoteAuthenticatorModules.getDefaultModule());
```

Comment:
I suspect we'll still need this module when the feature is disabled, right?

Reply:
It is getting added in else condition on line no 111. Am I missing something?

Reply:
Oh I missed the else condition. Thanks!